src/components/FilterTodos/FilterTodos.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-nested-ternary */
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Link, useLocation } from 'react-router-dom';
+import { ALL, FILTER_LINKS } from '../../constants';
+
+export const FilterTodos = ({ todos, setTodos }) => {
```

Comment:
```suggestion
export const FilterTodos = ({ todos = [], setTodos }) => {
```
instead of defaultProps at the end